src/System.Net.Libuv/src/project.json

```diff
@@ -17,6 +17,9 @@
     "System.Buffers": "0.1.0-*"
   },
   "frameworks": {
-    "dnxcore50": {}
```

Comment:
This is what we use across the board now. We should update whole sale. Let me know what to change it to, and I will submit a PR.